starboard/android/apk/app/src/main/java/dev/cobalt/coat/StarboardBridge.java

```diff
@@ -188,43 +187,44 @@ protected void onServiceDestroy(Service service) {
   @SuppressWarnings("unused")
   @UsedByNative
   protected void startMediaPlaybackService() {
-    if (cobaltMediaSession == null || !cobaltMediaSession.isActive()) {
```

Comment:
Maybe we could go a step further and make it a persistent setting that defaults to off ? Once we are ready to try it, it's already built in and can be enabled through JS.

Reply:
I don't think it has to be persistent, as long as it can be enabled from JS. It's (for now) unlikely we'll need a feature that can start background playback without first going in the foreground.

Reply:
That sounds reasonable - so the trick would only be plumbing the setting through from h5vcc down to Java code here.